packages/auth-providers/netlify/setup/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@redwoodjs/auth-netlify-setup",
+  "version": "3.2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/redwoodjs/redwood.git",
+    "directory": "packages/auth-providers/netlify/setup"
+  },
+  "license": "MIT",
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
```

Comment:
There's no `dist/index.js` file; this either needs to point to setup file in dist or we need to add an index like in auth0